pkg/worker/worker.go

```diff
@@ -60,9 +61,16 @@ func GatherResults(waitfile string, url string) error {
 		url += "." + filenameParts[1]
```

Comment:
line 45: inputFileName is misleading, it's not the name of a file, it's the contents of waitfile.